research/mobilenet/mobilenet_v1.py

```diff
@@ -0,0 +1,113 @@
+# Copyright 2020 The TensorFlow Authors. All Rights Reserved.
```

Comment:
Is there a reason for having mobilenet_v1/v2/v3.py if Config is capturing the differences? May be we can have one file for building different models.